app/Http/Controllers/Maps/MapGroupController.php

```diff
@@ -5,12 +5,41 @@
 use App\Facades\CampaignLocalization;
 use App\Http\Controllers\Controller;
 use App\Http\Requests\StoreMapGroup;
+use App\Facades\Datagrid;
+use App\Http\Requests\ReorderGroups;
+use App\Datagrids\Actions\GroupDatagridActions;
 use App\Models\Campaign;
 use App\Models\Map;
 use App\Models\MapGroup;
 
 class MapGroupController extends Controller
 {
+    /** @var string|null The datagrid controlling the bulk actions*/
+    protected $datagridActions = GroupDatagridActions::class;
+
+    /**
+     * Index
+     */
+    public function groups(Map $map)
+    {
+        $campaign = CampaignLocalization::getCampaign();
+        $options = ['map' => $map];
+        $model = $map;
+
+        Datagrid::layout(\App\Renderers\Layouts\Map\Group::class)
+            ->route('maps.groups', $options);
+        $rows = $map
```

Comment:
This can be improved :) 
![image](https://user-images.githubusercontent.com/3263440/196011006-280fd8e6-8dd7-4346-8da8-ad2b4199abb8.png)